kmk/modules/combos.py

```diff
@@ -56,7 +56,7 @@ def __repr__(self):
         if self._match_coord:
             return f'{self.__class__.__name__}({list(self.match)})'
         else:
-            return f'{self.__class__.__name__}({[k.code for k in self.match]})'
+            return f'{self.__class__.__name__}({[k.code if hasattr(k, 'code') else k for k in self.match]})'
```

Comment:
I was about to push a fix myself.
The conditional format for keys and coordinates was only there because the previous `Key.__repr__` was verbose to the point of unusability. We can drop all of that and make it:
```
def __repr__(self):
    return f'{self.__class__.__name__}({list(self.match)})'
```